nx-X11/include/Xos_r.h

```diff
@@ -476,15 +456,9 @@ typedef struct {
     (sp).sptr )
 
 #elif defined(XUSE_NETDB_R_API)
-/*
- * POSIX does not specify _r equivalents for <netdb.h> API, but some
- * vendors provide them anyway.  Use them only when explicitly asked.
- */
```

Comment:
Tricky rebasing now. I re-add the commit to the commit that removes __osf__ support, and then drop this comment with the complete _POSIX_REENTRANT_FUNCTIONS block again in a later commit. Is a bit to and fro, but this new way, it will be cleaner.